_posts/en/newsletters/2019-12-11-newsletter.md

```diff
@@ -0,0 +1,136 @@
+can make this a help-test-the-RC item -->
+
+- **Upgrade to LND 0.8.2-beta:** this [release][lnd 0.8.2-beta] contains
+  several bug fixes and minor UX improvements, most notably for the
+  recovery of static channel backups.  (One of these improvements is
+  [described][lnd-3698] later in this newsletter.)
+
+## News
+
+- **Watchtowers for eltoo payment channels:** [eltoo][topic eltoo] is a
+  proposed alternative payment channel layer for LN that doesn't require
+  participants be able to generate penalty transactions.
+  [Watchtowers][topic watchtowers] are services that broadcast a
+  pre-programmed transaction if they detect that one of their client's
+  channels is being closed using an older state; this allows their
+  clients to go offline without risking a loss of funds.
+
+    Conner Fromknecht started a [thread][fromknecht eltoo tower] asking
+    what data watchtowers would need to be store for eltoo and how that
+    would affect the scalability of watchtowers or the privacy of their
+    clients.  One option would be for a watchtower to store only the
+    latest update transaction.  This is highly scalabale because it only
+    requires a constant amount of storage per channel, and it's secure
+    because only the final settlement transaction can spend from the
+    final update transaction.  The offline node could broadcast that
+    settlement transaction whenever it next came online, even if that
+    were months or years later.
+
+    An alternative mechanism discussed would be for nodes to store the
+    settlement transaction with the watchtower.  This could provide
+    additional safety in case the node lost all data while it was
+    offline by sending funds to the node's desired withdrawal address
+    (which could be, for example, its cold wallet).  However, it would
+    increase the storage requirements for watchtowers and, worse, the
+    obvious way to implement it would significantly reduce user privacy
+    by giving watchtowers enough data to learn details about previous
+    payments made in the user's payment channels.  Some participants in
+    the thread discussed ways to obtain the safety benefits while
+    mitigating the privacy loss, although no clear conclusion was
+    reached in the thread as of this writing.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [C-Lightning #3260][] adds new `createonion` and `sendonion` RPC
+  methods that allow external tools or C-Lighting plugins to create and
+  send encrypted LN messages that the node itself doesn't necessarily
+  understand.  Some use cases for this mechanism described in the PR
+  include: cross-chain atomic swaps, rendezvous routing (see [Newsletter
```

Comment:
s/rendezvous/rendez-vous/?

Reply:
Almost always spelled without a hyphen in English, or at least American English.  See the examples here: https://en.wikipedia.org/wiki/Rendezvous

Reply:
Weird :smile: I checked the RFC before commenting and it's spelled correctly there https://github.com/lightningnetwork/lightning-rfc/wiki/Rendez-vous-mechanism-on-top-of-Sphinx

Reply:
One for the style guide? The BOLT uses 'rendez vous' in the title and 'rendez-vous' throughout the text. Newsletter 22 uses 'rendez-vous'.